collect_app/src/main/java/org/odk/collect/android/preferences/MapsPreferences.java

```diff
@@ -57,6 +64,23 @@ public static MapsPreferences newInstance(boolean adminMode) {
         return prefs;
     }
 
+    @SuppressLint("RestrictedApi")
```

Comment:
We should absolutely avoid suppressing lint warnings if they get in our way unless we really need to. Here it's telling us that we should be using this API as they might change it which would mean our code could end up breaking down the line. 

Not sure why we need to do this here and not use the XML attributes like on other preferences screens?

Reply:
@seadowg I shifted the `app:allowDividerBelow="false"` in the XML file, but I couldn't set the `icon_frame` visibility to `View.GONE` here, as the preferences are created dynamically for the `Basemap` Preference Category. I removed `SuppressLint` annotation though.

Reply:
Correct @seadowg! I fixed it. I set the `iconSpaceReserved` to false in the `MapPreference` class.